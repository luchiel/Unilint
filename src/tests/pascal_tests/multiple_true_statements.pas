multiple_true_statements
var
  i: integer;
begin
  for i := 1 to 2 do whiteln('oops');
  if a > b then writeln('a') else whiteln('b');
end.