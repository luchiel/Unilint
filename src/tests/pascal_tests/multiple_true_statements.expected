5:22: two or more expressions at one line
6:17: two or more expressions at one line
6:30: two or more expressions at one line
6:35: two or more expressions at one line
